src/pages/public/chat/components/ChatSidebar.tsx

```
@@ -0,0 +1,174 @@
+  SidebarHeader,
+  SidebarRail
+} from "@/components/ui/sidebar";
+import {
+  AudioWaveform,
+  BookOpen,
+  Bot,
+  Command,
+  Frame,
+  GalleryVerticalEnd,
+  LucideIcon,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)